dev-utils/poly-cli/src/templates/manifest.js

```diff
@@ -1,19 +1,38 @@
-export function manifestTemplate(feature_name, author) {
-    let t = {
-        name: feature_name,
-        author: author,
-        description: "",
-        thumbnail: "",
-        primaryColor: "#ffffff",
-        links: {},
-        translations: {
-            de: {
-                author: "",
-                description: "",
-                links: {},
+export function manifestTemplate(feature_name, author, feature_type) {
+    let t;
+    if (feature_type === "preview") {
+        t = {
```

Comment:
Nit: Any reason why `version` and `thumbnailColor` are not added?

Reply:
Will add them